package.json

```diff
@@ -130,6 +129,7 @@
     "react-native-permissions": "^2.0.10",
     "react-native-qrcode-svg": "^5.1.2",
     "react-native-randombytes": "^3.0.0",
+    "react-native-restart": "^0.0.17",
```

Comment:
RNRestart loads the bundle again and clears the JS memory (redux, class instances) completely, which loads all the JS instances again and loads the whole app with new instances. This ensures that the new instances will trigger the correct environment setup from storage, as well as restart all necessary startup components.

Reply:
This to me indicates that we might have not taken the correct approach in the first place in writing components, class instances etc if we're having to clear out the whole state as opposed to the components now reading from a different source. I appreciate that this is a bit of a hangover from a time when everything was stored locally in the app, but this isn't the case moving forward. This will need to be something we revisit in future from an architectural point of view.